### DISCUSSION TITLE
Set custom manager in serializer

### DISCUSSION THREAD

**Discussion by momen7899** [encode/django-rest-framework#9044] [Question & Answer]
Let me explain the problem I faced myself.
I have 2 managers in my model and I use this model in 2 APIs. Now I want to use one of these managers in each of the APIs.
Because I use my model as nested, I cannot specify which one of the managers to use in the view, and I need to do this in my nested serializer.
How can I do it?

**Comment by iamMHZ** [comment]
@momen7899 
Can you provide a code example?
